lib/module/interject.go

```diff
@@ -0,0 +1,27 @@
+package module
+
+import (
+	"fmt"
+	"github.com/dvdmuckle/irkbot/lib/configure"
+	"github.com/dvdmuckle/irkbot/lib/message"
+	"strings"
+)
+
+func HelpInterject() []string {
+	// This function returns an array of strings describing this command's
+	// functionality. It is displayed when someone gives the help command in a
+	// channel or private message.
+	s := "interject <subject> - return a popular copypasta defaulting to our favorite operating system"
```

Comment:
It should be `[subject]`, to indicate that the subject is optional.

Also, I'd change the description to: `give a pedantic rant about the proper way to refer to the given subject, defaulting to Linux if no subject given`. But that string is probably too long for one line, so please break it up using the `+` operator.